### PR TITLE
Remove exec.command checking

### LINKED ISSUE #760: Warn if added kubeconfig contains relative paths
Users are having issues on connecting to cluster if Kubeconfig (namely `exec`) contains non-absolute binary paths. Some shell environments might add something to `PATH` and that is not available when app is started from app menu. Lens could analyse added kubeconfigs and warn users if relative paths are found in Kubeconfig.

### LINKED ISSUE #878: can't open any clusters from kube config exec not found in $PATH
**Describe the bug**
When I try to open any of the clusters from the app, it can't authenticate saying that either `aws` or `aws-iam-authenticator` is not available in $PATH. However both are in my $PATH:

```
⋊> ~ ls /usr/local/bin/ | grep aws-iam                                                                                       16:20:00
aws-iam-authenticator
⋊> ~ ls /usr/local/bin/ | grep aws                                                                                           16:20:02
aws
aws-iam-authenticator
aws-shell
aws-shell-mkindex
aws-vault
aws_completer
aws_legacy_completer
rawshark
⋊> ~ echo $PATH | grep /usr/local/bin                                                                                        16:20:04
/Users/leepatrick/.fzf/bin /usr/local/bin
⋊> ~ which aws-iam-authenticator                                                                                             16:20:24
/usr/local/bin/aws-iam-authenticator
⋊> ~ which aws                                                                                                               16:21:21
/usr/local/bin/aws
```

**Environment (please complete the following information):**
- Lens Version: 3.6.0
- OS: OSX
- Installation method (e.g. snap or AppImage in Linux): brew
 
**Logs:**

```
2020/09/14 15:52:24 http: proxy error: getting credentials: exec: exec: "aws": executable file not found in $PATH

getting credentials: exec: exec: "aws": executable file not found in $PATH
----------------------
2020/09/14 16:15:57 http: proxy error: getting credentials: exec: exec: "aws-iam-authenticator": executable file not found in $PATH

getting credentials: exec: exec: "aws-iam-authenticator": executable file not found in $PATH

```

### LINKED ISSUE #2079: User Exec command "/usr/local/bin/aws" not found on host.
**Describe the bug**
I am using Lens on Windows, all my interactions with a kubenetes cluster i perform via kubectl installed on Windows Subsystem for Linux (WSL). I have also installed aws cli on WSL. My kubeconfig file is in WSL, i am able to reference the file in Lens using the path `\\wsl$\Ubuntu-18.04\home\user\.kube`
When i try to use one of my aws eks clusters, i get the above error message. I am assuming it's because it's trying to use aws cli from windows.

Is there any way to get it working without having to install aws cli in two places (WSL and Windows)?

**To Reproduce**
Use a kubeconfig file that contains a reference an aws eks cluster, see below.

The problem arises where the `command: /usr/local/bin/aws` has been defined, i added the fully qualified path but that hasn't helped.

**Expected behavior**
It should be able to load the cluster

**Environment (please complete the following information):**
- Lens Version: 4.0.8
- OS: Windows 10 Pro
- Installation method (e.g. snap or AppImage in Linux): From websiteusing installer
- WsL: Ubuntu 18.04

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
User Exec command "/usr/local/bin/aws" not found on host.
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: 
    server: https://asdfasdfasdfasdfasd.gr7.us-east-2.eks.amazonaws.com
  name: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
contexts:
- context:
    cluster: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
    namespace: xxxxxxx-rc5
    user: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
  name: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
current-context: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
kind: Config
preferences: {}
users:
- name: arn:aws:eks:us-east-2:999999999999:cluster/xxxxxxx-1
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - --region
      - us-east-2
      - eks
      - get-token
      - --cluster-name
      - xxxxxxx-1
      - --role
      - arn:aws:iam::999999999999:role/xxxxxxx-1-cluster-admin
      command: /usr/local/bin/aws
      env:
      - name: AWS_PROFILE
        value: xxxxxxx-eks
```

### LINKED ISSUE #3856: Last update 5.2.3-latest.20210921.1 broke AWS EKS kubernetes connectivity
**Describe the bug**
After upgrading to Lens: 5.2.3-latest.20210921.1 (don't remember which was the previous version)
- An error message appeared the first time I reloaded Lens mentioning plugins that no longer worked (I did not really pay attention to this message to remember it exactly)
- I lost all my configured clusters : they are greyed out on the right, unclickable, no right click option
Screenshot 2021-09-21 at 15 21 50
Screenshot 2021-09-21 at 15 22 21

- The sync kubernetes config button does nothing
- I am no longer able to configure & connect to my k8s clusters using aws iam authenticator

**To Reproduce**
My kubeconfig was similar to this one before upgrading

```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: XXXX
    server: https://XXXX.gr7.eu-central-1.eks.amazonaws.com
  name: eks_ENV
contexts:
- context:
    cluster: eks_ENV
    user: eks_ENV
  name: eks_ENV
current-context: eks_ENV
kind: Config
preferences: {}
users:
- name: eks_ENV
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - token
      - -i
      - XXX (ENV)
      command: aws-iam-authenticator
      env: null
```

After upgrading to aforementioned lens version, All config are greyed out and unmanageable

When I try to add again the same config, I have an error 
> ExecValidationNotFoundError: User Exec command "aws-iam-authenticator" not found on host. Please ensure binary is found in PATH or use absolute path to binary in Kubeconfig

After adding the absolute path (`/usr/local/bin/aws-iam-authenticator`), I can submit the kubeconfig but it is stuck on loading when I try to connect to a cluster

**Expected behavior**
Preserve existing kubeconfigs

**Screenshots**
Added inline above

**Environment (please complete the following information):**
- Lens Version: 5.2.3-latest.20210921.1
- OS: macOS
- Installation method (e.g. snap or AppImage in Linux): clicked auto-update update button

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
How do you do that on macOS ?
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: XXXX
    server: https://XXXX.gr7.eu-central-1.eks.amazonaws.com
  name: eks_ENV
contexts:
- context:
    cluster: eks_ENV
    user: eks_ENV
  name: eks_ENV
current-context: eks_ENV
kind: Config
preferences: {}
users:
- name: eks_ENV
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - token
      - -i
      - XXX (ENV)
      command: aws-iam-authenticator
      env: null
```

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

I narrowed into this as the solution to fixes https://github.com/lensapp/lens/issues/2079, fixes https://github.com/lensapp/lens/issues/878, and fixes https://github.com/lensapp/lens/issues/3856 by looking at the requirements as to why the check was introduced in the first place.

The check was introduced by https://github.com/lensapp/lens/pull/1013 to fix https://github.com/lensapp/lens/issues/760 and https://github.com/lensapp/lens/issues/654 and https://github.com/lensapp/lens/issues/653.

Now the original issue was about not knowing why a user could not connect to a cluster. The reason for that was that the executable wasn't found in the PATH env var. However, the PR to fix this issue instead of adding more visibility to what was going wrong, blocked users from adding the clusters in the first place.

In the current master of Lens, however, if you try and connect to a cluster and we cannot find the command that information is now displayed to the user:

image

The reason we should remove this check is that, especially on windows, it seems to be very flaky and at least one user who was experiencing this problem found that it was resolved (even by not changing anything to how they were running lens) by using a version of lens that did not have this check.

### CODE REVIEW

**Review by jim-docker — COMMENTED** [review]
Why is the "It looks like..." message shown 4 times (3 times in the scrollable field and once below it)?
Can that be reduced?
Screen Shot 2021-09-23 at 7 07 07 PM

**Comment by Nokel81** [comment]
@jim-docker I think that might be because the proxy process both emits an error to `stderr` and to the `error` event handler. I think we should merge this as is and maybe clean it up in the future.

File: src/common/kube-helpers.ts [hunk 1]
```diff
@@ -243,70 +251,25 @@ export function dumpConfigYaml(kubeConfig: Partial<KubeConfig>): string {
   return yaml.safeDump(config, { skipInvalid: true });
 }
 
-export function podHasIssues(pod: V1Pod) {
```
Comment (nevalla) [inline src/common/kube-helpers.ts:246]: these functions are not called anymore?

**Review by nevalla — APPROVED** [review]
Added comment about removed functions, not sure if those are dead code or not

File: src/common/kube-helpers.ts (hunk 1 above)
Comment (Nokel81) [reply src/common/kube-helpers.ts:246]: Nope and they are not exported either

**Pull request merged**